dali/operators/reader/loader/loader.cc

```diff
@@ -19,43 +19,59 @@ namespace dali {
-accuracy in some cases)code", false)
+      R"code(Determines whether the reader should stick to a data shard instead of going through
+the entire dataset.
+
+If decoder caching is used, it significantly reduces the amount of data to be cached, but
+might affect accuracy of the training.)code", false)
   .AddOptionalArg("read_ahead",
-      R"code(Whether accessed data should be read ahead. In case of big files like LMDB,
-RecordIO or TFRecord it will slow down first access but will decrease the time of all following
-accesses.)code", false)
+      R"code(Determines whether the accessed data should be read ahead.
+
+For large files such as LMDB, RecordIO, or TFRecord, this argument slows down the first access but
+decreases the time of all of the following accesses.)code", false)
   .AddOptionalArg("prefetch_queue_depth",
-      R"code(Specifies the number of batches prefetched by the internal Loader. To be increased when pipeline
-processing is CPU stage-bound, trading memory consumption for better interleaving with the Loader thread.)code", 1)
+      R"code(Specifies the number of batches to be prefetched by the internal Loader.
+
+This value should be increased when the pipeline is CPU-stage bound, trading memory
+consumption for better interleaving with the Loader thread.)code", 1)
   .AddOptionalArg("skip_cached_images",
-      R"code(If set to true, loading data will be skipped when the sample is present in the decoder cache.
-In such case the output of the loader will be empty)code", false)
+      R"code(If set to True, the loading data will be skipped when the sample is
+in the decoder cache.
+
+In this case, the output of the loader will be empty.)code", false)
   .AddOptionalArg("lazy_init",
-      R"code(If set to true, Loader will parse and prepare the dataset metadata only during the first `Run`
-instead of in the constructor.)code", false)
+            R"code(Parse and prepare the dataset metadata only during the first run instead of
+in the constructor.)code", false)
   .AddOptionalArg("pad_last_batch",
-      R"code(If set to true, the Loader will pad the last batch with the last image when the batch size is
-not aligned with the shard size. It means that the remainder of the batch or even the whole batch can be
-artificially added when the data set size is not equally divisible by the number of shards, and the shard is
-not equally divisible by the batch size. In the end, the shard size will be equalized between shards.)code", false)
+      R"code(If set to True, when the batch size is not aligned with the shard size, the Loader
+pads the last batch with the last image.
+
+The rest of the batch, or even an entire batch, can be added when the dataset size is not equally
+divisible by the number of shards, and the shard is not equally divisible by the batch size. The
+shard size will ultimately be equalized between shards.)code", false)
```

Comment:
```suggestion
      R"code(If set to True, pads the shard by repeating the last sample.

.. note::
  If the number of batches differs across shards, this option can cause an entire batch of repeated samples
  to be added to the dataset.)code", false)
```

Reply:
Does it make sense?

Reply:
It does, but I don't know if the user will get the full understanding what can happen there.

Reply:
Done